app/src/main/java/org/mozilla/fenix/compose/home/HomeSectionHeader.kt

```diff
@@ -105,17 +103,19 @@ private fun HomeSectionHeaderContent(
         )
 
         onShowAllClick?.let {
-            ClickableText(
-                text = AnnotatedString(text = stringResource(id = R.string.recent_tabs_show_all)),
-                modifier = Modifier.padding(start = 16.dp)
-                    .semantics {
-                        contentDescription = description
+            TextButton(
+                text = stringResource(id = R.string.recent_tabs_show_all),
+                buttonColors = textButtonColors(
+                    contentColor = if (isSystemInDarkTheme()) {
+                        PhotonColors.White
+                    } else {
+                        PhotonColors.Black
```

Comment:
Where are these color values coming from?

Reply:
I added these values to match the ripple effect applied to the button before the compose migration.

Reply:
Still just struggling a little bit with understanding this one. Could you point me to the code that we are matching the ripple effect to? For some reason, I generally assume the ripple effect is automatically provided by the built-in material composable components so I am still of the belief that these colors are built-in and provided for free since it's strange to see that we need to provide them. My secondary issue is the fact this wouldn't scale if we had more than the light/dark theme.

Reply:
I believe [this](https://github.com/mozilla-mobile/fenix/blob/e373f507a117679bf58e796834d7a4ebcf805c17/app/src/main/res/layout/recent_bookmarks_header.xml#L31) was the line responsible for adding the ripple effect. For composable buttons this is indeed built-in. In this case, if it wouldn't be specified, the ripple color would be picked up from the color of the text, which is `textAccent` (Violet20). If the ripple color before moving to compose is not the one we want to keep I can revert this change.